File: JavaStuff/Reference/WarmupDay2.java
public class WarmupDay2 {
	public static void main(String[] args) {
		int[] a;
		a = new int[3];
		a[0] = 10;
		a[1] = 5;
		a[2] = 7;
		int out = 0;
		int out2 = 9999999;
		for (int i = 0; i < a.length; i++) {
			if (a[i] > out) {
				out = a[i];
			}
			if (a[i] < out2) {
				out2 = a[i];
			}
		}
		System.out.println(out);
		System.out.println(out2);
	}
}

File: PythonStuff/Working/WarmupDay2.py
a = (10, 5, 7)
out2 = 999999999999999999999999999999
out = 0
for i in range(len(a)):
	if a[i] > out:
		out = a[i]
	if a[i] < out2:
		out2 = a[i]
print(out)
print(out2)
